### PR TITLE
configure application for heroku

### PR DESCRIPTION
**Pull request by SimonLab**
ref: https://github.com/dwyl/app/issues/231

- Setup phoenix application to work on Heroku by following the guide: https://hexdocs.pm/phoenix/heroku.html
- update Readme to add travis badge and describe which environment variable to define on localhost

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -440,6 +440,7 @@ see:
 
 To start your Phoenix server:
 
+  * Make sure you have Postgres installed, see: https://github.com/dwyl/learn-postgresql
```
Comment (nelsonic) [inline README.md]: 👍

**Comment by nelsonic** [comment]
@SimonLab this is looking good so far. 👍 
Do you need anything from me/us to make further progress? 💭 
Thanks! 😊

**Comment by SimonLab** [comment]
@nelsonic I think I'll review and merge #18 first then fix any merge conflicts. The last think I need to do otherwise on this PR is to add the `app.json` file configuration for Heroku review apps

**Comment by SimonLab** [comment]
The build is failing for this merged PR on heroku because the we have changed the structure of the database and the `status` table already exists:
image

On localhost we could run `mix ecto.reset` to reset the database (ie drop, create and migrate) however on Heroku we don't have the permission to drop the database:
image

image

However with the heroku cli we get
- 1. get the database name linked to the application:
```sh
heroku pg:info -a dwylapp
```
image

- 2. reset the database
```sh
heroku pg:reset postgresql-concentric-52258 -a dwylapp
```
image

- 3. run the migration again
```sh
heroku run MIX_ENV=prod mix ecto.migrate -a dwylapp
```
image

**Pull request merged**